src/services/ai-service/ai-service.types.ts

```diff
@@ -38,10 +38,13 @@ export interface WriteResponse {
 }
 
 export interface AIServiceError {
-  code: number;
+  type: AIServiceErrorTypes;
   message: string;
+  code: number;
 }
 
 export enum AIServiceErrorTypes {
```

Comment:
To update these types we need the ENUM list with the errors codes from the ai-service backendteam;
The ones added here are just for testing purposes